lib/js-file.js

```diff
@@ -102,6 +124,8 @@ JsFile.prototype = {
      */
     isEnabledRule: function(ruleName, line) {
         var enabled = true;
+        ruleName = ruleName.trim();
```

Comment:
(Off the top of my head) This was because something was returning rule names with surrounding whitespace, and I think that rule names should never have whitespace.

I do not know if this is actually required for correct behaviour currently, it could have been a hang over from a previous state. I'll check when I next have access to a computer, which is probably Monday.